chart/templates/openshift/route.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.route.enabled -}}
```

Comment:
I would move this directly into the templates folder personally, but that is just personal preference.